content/en/docs/installation/upgrading/remove-deprecated-apis.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Remove deprecated API resources"
+linkTitle: "Remove deprecated API resources"
```

Comment:
nitpick: we currently use title casing everywhere for the sidebar. Also, does this sound better with "removing" instead of "remove", to indicate it's a guide?

```suggestion
title: "Removing Deprecated API Resources"
linkTitle: "Removing Deprecated API Resources"
```

Reply:
Ahh, yes the title case :)

Reply:
:see_no_evil: I'm sorry :see_no_evil: